### ISSUE TITLE
Support Java 8

### ISSUE THREAD

**Issue by sebplorenz** [openshift/wildfly-8-centos#41]
I'm currently using the following script to create a docker image that inherits from this image and has Oracle Java 8 installed:
```
# Install wget and the Development Tools to be able to download SWI Prolog and
# run the installer
RUN yum install -y wget && \
    yum clean all -y
    
# Install Java 8
RUN cd /opt && \
    wget --no-cookies --no-check-certificate --header "Cookie: gpw_e24=http%3A%2F%2Fwww.oracle.com%2F; oraclelicense=accept-securebackup-cookie" \
 "http://download.oracle.com/otn-pub/java/jdk/8u45-b14/jre-8u45-linux-x64.tar.gz" && \
   tar xzf jre-8u45-linux-x64.tar.gz && \
   alternatives --install /usr/bin/java java /opt/jre1.8.0_45/bin/java 2 && \
   alternatives --set java  /opt/jre1.8.0_45/bin/java && \
   rm -f jre-8u45-linux-x64.tar.gz
...
```
But this may bring licensing issues when publicly distributed on docker hub. There are no packages for OpenJDK 8 on centos 7 at the moment. Can the packages from centos 6 be used in this image?

**Comment by bparees** [comment]
I don't know, I wouldn't recommend it, but feel free to give it a try.

**Comment by sebplorenz** [comment]
Ok, there are openjdk 1.8 packages available for centos 7. I will try them.

**Comment by soltysh** [comment]
Closing, since this was resolved in #42.
